### PR TITLE
Feature/27 google maps integrate spring boot

### LINKED ISSUE #27: Research google maps API and integrate it with BE spring boot application

### PR DESCRIPTION
**Pull request by StoyanTinchev**
### Close #27 
* Added:
  * Google maps example integration
  * [Introduce GoogleMapsService to MessageService.
  * develop merge
  * Google Maps service logic

### CODE REVIEW

File: platform/comm/src/main/java/com/fmicodes/comm/DTO/RestaurantInfo.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+public class RestaurantInfo {
+    private String name;
+    private String photoUrl;
+    private String location;
+    private double rating;
+    private Integer priceLevel;
+    private int userRatingsTotal;
+    private String openingHours;
+    private String googleMapsLink;
```
Comment (AntonioK180) [inline platform/comm/src/main/java/com/fmicodes/comm/DTO/RestaurantInfo.java]: change the primitive types with their corresponging objects, e.g. int -> Integer

File: platform/comm/src/main/java/com/fmicodes/comm/services/GoogleMapsService.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                            String photoReference = photo.get("photo_reference").asText();
+                            restaurant.setPhotoUrl(getPhotoUrl(photoReference, photoHeight, photoWidth));
+                        }
+                    }
+
+                    restaurants.add(restaurant);
+                }
+            }
+
+            return restaurants;
```
Comment (AntonioK180) [inline platform/comm/src/main/java/com/fmicodes/comm/services/GoogleMapsService.java]: Move return out of try block and at the end of the method

**Review by AntonioK180 — CHANGES_REQUESTED** [review]

File: platform/comm/src/main/java/com/fmicodes/comm/DTO/RestaurantInfo.java (hunk 1 above)
Comment (StoyanTinchev) [reply platform/comm/src/main/java/com/fmicodes/comm/DTO/RestaurantInfo.java]: taken into account

File: platform/comm/src/main/java/com/fmicodes/comm/services/GoogleMapsService.java (hunk 2 above)
Comment (StoyanTinchev) [reply platform/comm/src/main/java/com/fmicodes/comm/services/GoogleMapsService.java]: taken into account

**Review by AntonioK180 — APPROVED** [review]

**Pull request merged**